src/app/fsat/fsat-report/detailed-results/detailed-results.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FanAnalysisService } from '../../../calculator/fans/fan-analysis/fan-analysis.service';
+import { FsatReportRollupService } from '../../../report-rollup/fsat-report-rollup.service';
+import { Assessment } from '../../../shared/models/assessment';
+import { FSAT, PsychrometricResults } from '../../../shared/models/fans';
+import { Settings } from '../../../shared/models/settings';
+import { FsatService } from '../../fsat.service';
+
+@Component({
+  selector: 'app-detailed-results',
+  templateUrl: './detailed-results.component.html',
+  styleUrls: ['./detailed-results.component.css']
+})
+export class DetailedResultsComponent implements OnInit {
+
+  @Input()
+  settings: Settings;
+  @Input()
+  inRollup: boolean;
+  @Input()
+  assessment: Assessment;
+  selectedModificationIndex: number;
+  fsat: FSAT;
+  psychrometricResults: PsychrometricResults;
+
+  pressureCalcResultType: string = 'static';
+  
+
+  constructor(private fsatReportRollupService: FsatReportRollupService, private fsatService: FsatService,
+    private fanAnalysisService: FanAnalysisService) { }
+
+  ngOnInit(): void {
+    this.fsat = this.assessment.fsat;
+    if (this.inRollup) {
+      this.fsatReportRollupService.selectedFsats.forEach(val => {
+        if (val) {
+          val.forEach(assessment => {
+            if (assessment.assessmentId === this.assessment.id) {
+              this.selectedModificationIndex = assessment.selectedIndex;
+            }
+          });
+        }
+      });
+    }
+
+
+  }
+
+  getPsychrometricResults(fsat: FSAT): PsychrometricResults {
```

Comment:
This method can be moved to fsat-service. Use a similar pattern to how the planeResults are retrieved and set on the fsat.outputs. I have the original screen shot of this - let me know if you want to see or demo. Mark already mentioned but, you won't need to call either of these in this component ts or html, because the fsat object is passed down to these components.